cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/cats/sql/cluster/SqlClusteredAgentScheduler.kt

```diff
@@ -90,6 +92,11 @@ class SqlClusteredAgentScheduler(
     val lockInterval = agentLockAcquisitionIntervalSeconds ?: 1L
     lockPollingScheduler.scheduleAtFixedRate(this, 0, lockInterval, TimeUnit.SECONDS)
     enabledAgents = Pattern.compile(enabledAgentPattern, CASE_INSENSITIVE)
+    disabledAgents = dynamicConfigService.getConfig(
```

Comment:
I think this will require a clouddriver restart to pick up changes to the dynamic property, not sure if that's the intent?

Reply:
That's not the intent - good catch.  Let me fix that.